packages/paste-core/components/combobox/src/Combobox.tsx

```diff
@@ -38,10 +43,11 @@ export interface ComboboxProps extends Omit<FormInputProps, 'id' | 'type'> {
   onIsOpenChange?: UseComboboxPrimitiveProps<any>['onIsOpenChange'];
   onSelectedItemChange?: UseComboboxPrimitiveProps<any>['onSelectedItemChange'];
   optionTemplate?: (item: string | {}) => React.ReactNode;
+  selectedItem?: UseComboboxPrimitiveProps<any>['selectedItem'];
```

Comment:
Just curious why are these all typed to 'any' 🤔

Reply:
I _think_ it looks like we can remove them. I was just following the pattern, and I think we copy and pasted from Typescript

Reply:
Apparently not, but I have a way to at least make it more explicit.